exercises/queen-attack/queen-attack.spec.js

```diff
@@ -18,6 +18,7 @@ describe('Queens', function () {
 
     try {
       var queens = new Queens(positioning);
+      expect(queens.prop).toEqual([2, 4]);
```

Comment:
What is `prop`?  I'm guessing this works because an exception gets thrown on the prior line and this line of code is never reached.